library/globals.inc.php

```diff
@@ -1232,6 +1232,41 @@
       xl('Number of Appointments to display in the Onsite Patient Portal')
     ),
 
+    'appt_display_sets_option' => array(
+      xl('Appointment Display Sets - Ignore Display Limit (Last Set)'),
+      'bool',                           // data type
+      '1',                              // default
+      xl('Appointment Display Sets - Allow Last Set to Expand Past Appointment Display Limit')
```

Comment:
Do you mean will always shows all appt on same day even if over limit. Or mean something else?